### PR TITLE
try circleci

### PR DESCRIPTION
**Pull request by andreaseger**
Felt like we might as well switch this repo to circleci

### CODE REVIEW

**Comment by andreaseger** [comment]
ok we now just need to remove travisci from this repo

File: .circleci/config.yml [hunk 1]
```diff
@@ -0,0 +1,75 @@
+# Ruby CircleCI 2.0 configuration file
+#
+# Check https://circleci.com/docs/2.0/language-ruby/ for more details
+#
+
+common_steps: &common_steps
+  - checkout
+  - run: gem update --system
+  - restore_cache:
+      key: gem-cache-{{ .Branch }}-{{ checksum "Gemfile" }}
```
Comment (alexJunger) [inline .circleci/config.yml]: I guess this should include the gemspec checksum as well, if possible?

Comment (andreaseger) [reply .circleci/config.yml]: Ah true, using the gemspec makes more sense here. The original snippet used the gemfile.lock which doesn't work for gems. Will change that.

Comment (alexJunger) [reply .circleci/config.yml]: I guess using both would be best? If that's possible. Not sure how this caching works exactly on circleci, but modifying Gemfile without cache invalidation sounds prone to issues.

Comment (andreaseger) [reply .circleci/config.yml]: it's just the gem cache so IMHO the gemspec alone is enough

**Pull request merged**